v2/pkg/common/annotations/annotations.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package annotations
```

Comment:
minor: Consider putting each annotation + value pair into its own file. So this could be called `reconcile_policy.go`